core/players-handbook-2024/classes/class-cleric.xml

```diff
@@ -0,0 +1,447 @@
+			<stat name="cleric:spellcasting:slots:6" value="1" level="19" />
+			<stat name="cleric:spellcasting:slots:7" value="1" level="13" />
+			<stat name="cleric:spellcasting:slots:7" value="1" level="20" />
+			<stat name="cleric:spellcasting:slots:8" value="1" level="15" />
+			<stat name="cleric:spellcasting:slots:9" value="1" level="17" />
+
+			<stat name="cleric:spellcasting:prepare" value="4" level="1" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="2" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="3" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="4" />
+			<stat name="cleric:spellcasting:prepare" value="2" level="5" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="6" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="7" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="8" />
+			<stat name="cleric:spellcasting:prepare" value="2" level="9" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="10" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="11" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="13" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="15" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="17" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="18" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="19" />
+			<stat name="cleric:spellcasting:prepare" value="1" level="20" />
+
+
+			<select type="Spell" name="Cantrip (Cleric)" supports="$(spellcasting:list), 0" level="1" number="3" spellcasting="Cleric" />
+			<select type="Spell" name="Cantrip (Cleric)" supports="$(spellcasting:list), 0" level="4" number="1" spellcasting="Cleric" />
+			<select type="Spell" name="Cantrip (Cleric)" supports="$(spellcasting:list), 0" level="10" number="1" spellcasting="Cleric" />
+		</rules>
+	</element>
+
+	<element name="Level 1: Divine Order" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_DIVINEORDER">
+		<description>
+			<p>You have dedicated yourself to one of the following sacred roles of your choice. You may choose one of the following features to gain:</p>
+			<p class="indent"><i>Protector</i> or <i>Thaumaturge</i></p>
+		</description>
+		<sheet alt="Divine Order" display="false" />
+		<rules>
+			<select type="Class Feature" name="Divine Domain" supports="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_DIVINEORDER_PROTECTOR|ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_DIVINEORDER_THAUMATURGE" />
+		</rules>
+	</element>
+
+	<element name="Protector" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_DIVINEORDER_PROTECTOR">
+		<description>
+			<p class="indent"> <i>Protector.</i> Trained for battle, you gain proiciency with Martial weapons and training with Heavy armor.</p>
+		</description>
+		<sheet display="false" />
```

Comment:
I adopted the previous Aurora style formatting (see 2014 Life Cleric & Nature Cleric feature elements) where features which were already otherwise displayed on the sheet (in this case, under the Armor proficiencies) didn't get displayed in writing under features to minimize unnecessary text blobs